test/functional/commands/keyboard-e2e-specs.js

```diff
@@ -22,9 +22,15 @@ describe('createSession', function () {
   });
 
   it('should start android session focusing on default activity', async () => {
-    let el = await driver.elementById('edit1');
+    let el = await driver.elementById('id/edit1');
```

Comment:
Yes this was, again, just a POC and we wanted to get an actual functional test in. Once we come back to this and start work on it, if indeed we ever do, these things can be addressed.